components/cam/src/physics/rrtmgp/radiation.F90

```diff
@@ -1618,10 +1620,6 @@ subroutine radiation_driver_lw(state, pbuf, cam_in, is_cmip6_volc, &
             ))
             call t_stopf('rad_calculations_lw')
 
-            ! Check stuff
```

Comment:
Are these checks somewhere else? Or are they not valid?

Reply:
These were debug checks I had inserted at one point when trying to track down some NaN's in a history write. I removed them here because I didn't think they were really necessary for production runs.